Campus_Energy_Dataset_Doc.ipynb

```diff
@@ -7,44 +7,61 @@
    },
```

Comment:
<p>Can we remove the empty bullet under Part 1, and remove the extra space between the words "of" and "Electrcity" in 1a please? And thank you for adding in a link to the Additional Data Sources section, and putting in the cool example code!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yup, I'll fix this</p>

Reply:
fixed